suggestions/rss_readers.json

```diff
@@ -4,5 +4,19 @@
         "label": "Feeder",
         "source": "fg",
         "repo": "https://gitlab.com/spacecowboy/Feeder"
+    },
+    {
+        "id": "me.ash.reader",
+        "label": "Read You",
+        "source": "f",
+        "reason": "A modern and elegant RSS reader with Material You design",
+        "repo": "https://github.com/Ashinch/ReadYou"
+    },
```

Comment:
Feeder is chosen over Read You because Feeder offer better features (UX-UI trade-off).